packages/block-library/src/post-content/index.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-content` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-content` block on the server.
+ *
+ * @return string Returns the filtered post content of the current post.
+ */
+function render_block_core_post_content() {
+	rewind_posts();
+	the_post();
+	return apply_filters( 'the_content', str_replace( ']]>', ']]&gt;', get_the_content() ) );
```

Comment:
Should this not receive a wrapping element (like many themes put a `div.entry-content` around it)?

Reply:
Yeah, I've seen that, but I'm not very familiar with everything it's used for.

Is that something we should hardcode or expect themes to add it in a filter?

Reply:
I think we would need to align on a wrapping `div` with a certain class name. I assume based on what I've seen, one of the most common classes used for this is `entry-content`. An alternative would be to use a typical block class, e.g. `wp-block-post-content`.

This leads to a broader discussion on how the class naming scheme of Gutenberg should evolve when it comes to entire templates. Is the `wp-block-...` naming sufficient, should it expand to template blocks as well? Or should we constrain the typical block classes to in-content blocks and define a set of more traditional class names?

Finding a consistent solution here is crucial for themes styling Gutenberg templates.

Reply:
We'll probably have to use `entry-content` for backwards compatibility.

Reply:
I think that depends on whether we can get the approach in its entirety to be backward-compatible or not. I think a site-wide block based editing experience will require massive changes in how themes are developed either way. It shouldn't break existing themes, but I think certain features would just not be supported with an old theme.

Reply:
Yeah, but keeping it as `entry-content` wouldn't hurt.

Reply:
Yes, any existing class names from Microformats should be preserved, as they relate to semantics. And [`entry-content`](http://microformats.org/wiki/hatom#Entry_Content) is one such example from hAtom. Use of such legacy Microformats should not preclude using Schema.org microdata.